api-connection.html

```diff
@@ -100,7 +100,203 @@ <h2 id="function_newConnection">new Connection(config)</h2>
       <dd>
         Type of the authentication method, valid types are <code>default</code>, <code>ntlm</code>,
         <code>azure-active-directory-password</code>, <code>azure-active-directory-access-token</code>, 
-        <code>azure-active-directory-msi-vm</code>, or <code>azure-active-directory-msi-app-service</code> 
+        <code>azure-active-directory-msi-vm</code>, <code>azure-active-directory-service-principal-secret</code>,
+        or <code>azure-active-directory-msi-app-service</code> 
+        <ul>
+          <li><code>azure-active-directory-password</code></li>
+{% highlight javascript %}
+"authentication": {
+  ...,
+  "options": {
+    "userName": value;
+    "password": value;
+    "domain": value; (Optional)
+  }
+}           
+{% endhighlight %}
+          <ul>
+            <li>username (string): A user need to provide `userName` associated to their account.</li>
+            <li>password (string): A user need to provide `password` associated to their account.</li>
+            <li>domain (string): Optional parameter for specific Azure tenant ID.</li>
+          </ul>
+          <li><code>azure-active-directory-access-token</code></li>
+{% highlight javascript %}
+"authentication": {
+  ...,
```

Comment:
```suggestion
  ...,
  "type": "azure-active-directory-access-token",
```
Maybe add the type to each example config for clarity?